app/models/infrastructure_facades/bash_command.rb

```diff
@@ -0,0 +1,88 @@
+class BashCommand
+  attr_reader :commands
+
+  def initialize
+    @commands = []
+  end
+
+  def to_s
+    "/bin/bash -i -c '#{@commands.join(';')}'"
+  end
+
+  def to_raw_s
+    @commands.join(';')
+  end
+
+  def append(command)
+    if command.instance_of?(BashCommand)
+      @commands += command.commands
+    else
+      @commands << command
+    end
+
+    self
+  end
+
+  def mute(command)
+    log command, '/dev/null'
```

Comment:
Zgodnie z zaleceniami stylu: https://github.com/bbatsov/ruby-style-guide#no-dsl-parens
tutaj preferujemy ``()`` - chyba, że uznamy, że to DSL